pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -632,12 +633,16 @@ func (p *PriorityQueue) MoveAllToActiveQueue() {
 		} else {
 			if err := p.activeQ.Add(pInfo); err != nil {
 				klog.Errorf("Error adding pod %v to the scheduling queue: %v", pod.Name, err)
+			} else {
+				moved = true
```

Comment:
Instead you could call p.cond.Signal() here and delete the Broadcast below.

Reply:
I assume the author of p.cond.Broadcast() call would think broadcasting is better compared to signaling inside the loop.